corehq/apps/receiverwrapper/util.py

```diff
@@ -143,20 +133,11 @@ def get_build_version(xform):
     return None, BuildVersionSource.NONE
 
 
+@quickcache(['domain', 'build_or_app_id'], timeout=24*60*60)
 def get_app_and_build_ids(domain, build_or_app_id):
-    if build_or_app_id:
-        cache_key = 'build_to_app_id' + build_or_app_id
-        cache_value = cache.get(cache_key)
-        if cache_value is None:
-            cache_value = _get_app_and_build_ids(domain, build_or_app_id)
-            cache.set(cache_key, cache_value, 24*60*60)
-        return cache_value
-    else:
-        app_id, build_id = build_or_app_id, None
-    return app_id, build_id
-
+    if not build_or_app_id:
+        return build_or_app_id, None
```

Comment:
Is this always expected to return `None, None` or is `build_or_app_id` sometimes other falsy values like `''` or `0` etc?